app/frontend/src/cljs/teet/project/project_layers.cljs

```diff
@@ -173,20 +187,26 @@
 (defn- ags-on-select [e! {:map/keys [teet-id]}]
   (e! (map-controller/->FetchOverlayForEntityFeature [:route :project :overlays] teet-id)))
 
-(defn selected-cadastral-units [{{checked-cadastral-geojson :checked-cadastral-geojson
+(defn selected-cadastral-units [{{:keys [query]} :app
+                                 {checked-cadastral-geojson :checked-cadastral-geojson
                                   setup-step :setup-step} :project}]
   (when (and checked-cadastral-geojson
-             (= setup-step "cadastral-units"))
+             (or
+               (= (:configure query) "cadastral-units")
+               (= setup-step "cadastral-units")))
     {:checked-cadastral-geojson
      (map-layers/geojson-data-layer "selected-cadastral-units"
                                     (->js (assoc {"type" "FeatureCollection"} "features" (into [] checked-cadastral-geojson)))
                                     map-features/selected-cadastral-unit-style
                                     {:opacity 1})}))
 
-(defn selected-restrictions [{{checked-restrictions-geojson :checked-restrictions-geojson
+(defn selected-restrictions [{{:keys [query]} :app
+                              {checked-restrictions-geojson :checked-restrictions-geojson
                                setup-step :setup-step} :project}]
   (when (and checked-restrictions-geojson
-             (= setup-step "restrictions"))
+             (or
+               (= (:configure query) "restrictions")
+               (= setup-step "restrictions")))
```

Comment:
This pattern is repeated quite a few times. I guess this has to do with getting things working in both the wizard and the new views?

Reply:
Yes, would be great if this could somehow be made more clear.